src/models/chart-model/format-pattern-from-range.js

```diff
@@ -0,0 +1,51 @@
+import KEYS from '../../constants/keys';
+
+export default function getFormatPatternFromRange(scaleName, viewHandler, layoutService) {
+  let min;
+  let max;
+  if (scaleName === KEYS.SCALE.X) {
+    ({ xAxisMin: min, xAxisMax: max } = viewHandler.getMeta().homeStateDataView);
+  } else {
+    ({ yAxisMin: min, yAxisMax: max } = viewHandler.getMeta().homeStateDataView);
+  }
+  if (!min || !max) {
+    return false;
+  }
```

Comment:
should it return false here when min or max is 0?

Reply:
This is a silly mistake to make.
I will to change it to ` if (min !== undefined || max !== undefined) {
    return false;
  }`. 
  Do you have a better suggestion?

Reply:
can min or max be null? or is it only undefined that you need to test for.
(other values that could be relevant is NaN and Infinity)

Reply:
It seems that test function `Number.isFinite` will work, as it only returns `true` when min/max is an actual number.